Formula/rpki-client.rb

```diff
@@ -0,0 +1,49 @@
+class RpkiClient < Formula
+  desc "OpenBSD portable rpki-client"
+  homepage "https://github.com/rpki-client/rpki-client-portable"
+  url "https://github.com/rpki-client/rpki-client-portable/archive/refs/tags/7.4.tar.gz"
+  sha256 "cacb56107b29baef67c0117bc53d726c95684ad27a6affe2a1527470c67f5e86"
+  license "ISC"
+
+  depends_on arch: :intel
```

Comment:
```suggestion
  depends_on arch: :x86_64
```
Why doesn't this work on ARM?

Reply:
I don't have access to an M1Pro system and it failed test builds. I'd rather not speculate, I will have an M1pro system in the next 4-6 weeks